build.sbt

```diff
@@ -78,6 +78,10 @@ lazy val ops = project
     name := "ammonite-ops"
   )
 
+lazy val prompt = shellPrompt in ThisBuild := { state =>
```

Comment:
Sorry I should have been clearer. When you have a multi-module sbt project you can "navigate" from the top-level module to a submodule and only run the tests for that module:
```
./sbt
> project repl
> test
```

With this setting you will see the module name before the prompt so you know where you are
```
./sbt
> project repl
repl> test
```